cmake/modules/AddCephTest.cmake

```diff
@@ -124,6 +124,18 @@ function(add_tox_test name)
               --venv-path ${venv_path})
   set_tests_properties(${test_name} PROPERTIES
     FIXTURES_REQUIRED venv-for-${name})
+  set(toxtest_default_labels "PurePython;PythonTox")
```

Comment:
Is there something in the nature of tests which are added with run_tox_tests() that ensures they are pure Python?

Reply:
Not strictly, but the tox environment (by default) isolates the packages so that any C/C++/Cython extensions created by the build would need special casing to be included. Thus I think it's safe to label these that way at this time.  If in the future some of the tox tests started to need to "break out" of the isolated venv's to use the compiled extensions we could change the labeling.